lib/ansible/modules/cloud/vmware/vmware_host.py

```diff
@@ -199,17 +279,20 @@ def main():
     argument_spec = vmware_argument_spec()
     argument_spec.update(
         datacenter_name=dict(type='str', required=True),
-        cluster_name=dict(type='str'),
+        cluster_name=dict(type='str', required=True),
```

Comment:
Modify the documentation to reflect this change. Also, add version in "2.5" this behaviour is changed.

Reply:
done